apps/platform-app/src/app/[locale]/(moderation)/_actions/mutations/approveProfile.ts

```diff
@@ -19,12 +20,14 @@ export const approveProfile = withSentry(async (profileId: string) => {
     state: PublishingState.APPROVED,
   })
 
-  const profileOwnerUsername = await findGithubUsernameByProfileId(profileId)
+  const profileOwnerUsername =
+    ((await findGithubUsernameByProfileId(profileId)) as string) ??
+    ((await findLinkedinUsernameByProfileId(profileId)) as string)
 
   await sendProfileApprovedEmail(
     updatedProfile.user.email,
     profileOwnerUsername,
-  )
+  ) // does not open profile from email button. template changed? CC: Sumick
```

Comment:
@Sumick

Reply:
it works on prod - ensure you have latest changes because @Wiecek-K refactored it lately